### PR TITLE
Work for #2090 - added fonts

### PR DESCRIPTION
**Pull request by ST-1580**

### CODE REVIEW

**Review by tsv2013 — CHANGES_REQUESTED** [review]
I believe we needn't load all these fonts. @novikov82 will explain how it should be.

**Review by tsv2013 — CHANGES_REQUESTED** [review]
Sorry for the late feedback. It hasn't be ready in my mind.

**Comment by ST-1580** [comment]
Firstly, see this PR: https://github.com/surveyjs/survey-library/pull/3626

File: packages/survey-creator-core/src/variables.scss [hunk 1]
```diff
@@ -34,7 +34,7 @@ $green: var(--green, #19b394);
 $green-light: var(--green-light, rgba(25, 179, 148, 0.1));
 $blue-light: var(--blue-light, rgba(67, 127, 217, 0.1));
 
-$font-family: "Open Sans";
+$font-family: "Open Sans", "Roboto", sans-serif;
```
Comment (tsv2013) [inline packages/survey-creator-core/src/variables.scss:37]: "Roboto" is non-standard. We need to use: "Helvetica Neue", Helvetica, Arial, sans-serif
After open sans

**Review by tsv2013 — CHANGES_REQUESTED** [review]
We need to change fonts list

**Review by tsv2013 — APPROVED** [review]

**Pull request merged**